lib/python/treadmill_aws/cli/__init__.py

```diff
@@ -70,6 +70,18 @@ def handle_context_opt(ctx, param, value):
 
 # Suppress pylint warnings concerning unused Click arguments
 # pylint: disable=W0613
+def sanitize_user_name(ctx, param, value):
+    """Check usernames for length and invalid characters.
```

Comment:
if you are not using args, you can del them.

def foo(x):
   del x

pylint will not complain.

or start then with _.